CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Fixed
 - Handle missing target files without raising an exception (#4462)
+- Match resources in Java try-with-resources statements (#4228)
```

Comment:
Just spotted this when I was going to do the release, but note that you added the entry in the wrong place.